bridge/contracts/HecBridgeSplitter.sol

```diff
@@ -5,84 +5,66 @@ import '@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol';
 import '@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol';
 import '@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol';
 import '@openzeppelin/contracts-upgradeable/utils/math/SafeMathUpgradeable.sol';
+import {OwnableUpgradeable} from '@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol';
+import '@openzeppelin/contracts/utils/structs/EnumerableSet.sol';
 
-interface IOwnableUpgradeable {
-	function owner() external view returns (address);
+error INVALID_PARAM();
+error INVALID_ADDRESS();
+error INVALID_AMOUNT();
+error INVALID_ALLOWANCE();
+error INVALID_DAO_FEE();
+error INVALID_FEES();
+error DAO_FEE_FAILED();
 
-	function renounceManagement() external;
-
-	function pushManagement(address newOwner_) external;
-
-	function pullManagement() external;
-}
-
-abstract contract OwnableUpgradeable is IOwnableUpgradeable, Initializable, ContextUpgradeable {
-	address internal _owner;
-	address internal _newOwner;
+/**
+ * @title HecBridgeSplitter
+ */
+contract HecBridgeSplitter is OwnableUpgradeable, PausableUpgradeable {
+	using SafeMathUpgradeable for uint256;
```

Comment:
is there any need to use the `SafeMath` library?

Reply:
@xiangdev1016, as we are using solidity v0.8.0+, there's no need to use the SafeMath
Also, you are not using the add(), sub(), mul(), div() functions on the existing contract
So you can remove it